### ISSUE TITLE
Can you translate/tilt a mirror in poppy.OpticalSystem Class?

### ISSUE THREAD

**Issue by gkstmdwn** [spacetelescope/poppy#523]
I want to see how translating/tilting a mirror affects the PSF/WFE in an Optical System with more than 1 mirrors

**Comment by mperrin** [comment]
Yes, in general these can be modeled, though the details of how best to do so depends on the specific optical element class being used. Many of the optical element classes support translation and rotation via setting attributes `optic.shift_x` or `.shift_y` or `.rotation` (for rotation around the optical axis). For tilts, you can use the TiltOpticalPathDifference class, or the  TipTiltStage class to represent tilting some arbitrary optic (see https://poppy-optics.readthedocs.io/en/latest/available_optics.html#Tip-Tilt-Stage)
